.github/runlint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright (c) 2018, 2021 IBM Corp. and others
```

Comment:
If this is a new file, you can update 2018 to 2021.

Reply:
Updated. Thanks.